### PR TITLE
README: Describe the use of hwraid_hp_repository_version

### LINKED ISSUE #2: Debian stretch: hpacucli is no more, use ssacli
I've given a look at the repository structure of HPE's mcp repository for Debian stretch and following things have change from what I remember from jessie:

* hpacucli is no more, it has been replaced by ssacli
* hpssacli is now called ssacli
* ssa (Smart Storage Administration Utility) is now available like already referenced in RedHat.yml. It may have already been there with jessie, haven't check though.

Proposal: Either update Debian.yml to acommodate for Debian 9 (and add Debian_8.yml) or add Debian_9.yml. Personally I'd opt for an updated Debian.yml and add a Debian_8.yml instead.

What's your take?

### PR DESCRIPTION
**Pull request by matsimon**
##### SUMMARY

* Initial proposal on how to document variables
* Document hwraid_hp_repository_version

##### ISSUE TYPE

 - Docs Pull Request

##### ANSIBLE VERSION

```
$ ansible --version
ansible 2.5.2
  config file = /etc/ansible/ansible.cfg
  configured module search path = [removed]
  ansible python module location = /usr/lib/python2.7/dist-packages/ansible
  executable location = /usr/bin/ansible
  python version = 2.7.13 (default, Nov 24 2017, 17:33:09) [GCC 6.3.0 20170516]

```

##### ADDITIONAL INFORMATION

If accepted, resolves #2 

There is an open issue with the formatting of the URL to HPE's documentation: pandoc converts it right whereas Github doesn't seem to like it. In the end Galaxy needs to show it correctly.

Please let me know how it should be done.

### CODE REVIEW

**Comment by keachi** [comment]
You should document the variable in the `defaults/main.yml` because our CI process will document that in our documentation.

It would be nice to have that too in the readme, because Ansible Galaxy uses that, but I would recommend to build it in the CI.

**Comment by matsimon** [comment]
TL;DR as discussed p2p:
* Ansible galaxy uses README.rst which is likely what most people will check out when looking at it on Ansible Galaxy
* However docs.adfinis-sygroup.ch/public is the place for the CI-/auto-generated Documentation which isn't very well known outside of Adfinis SyGroup

We have to decide where documentation has to be made and make it clear in (unfortunately) all READMEs of our modules. That is either document both (that is README.rst and defaults/main.yml) or either point one of them to the right documentation.

Interestingly on our Docs page [Docs » ROLE OVERVIEW » ROLE HWRAID](https://docs.adfinis-sygroup.ch/public/ansible-guide/adfinis-sygroup.hwraid.html) it looks very much like README.rst ist being displayed.

@keachi You seem more familiar with the CI process. Could you tell me if whant I'm seeing is actually happening?

**Comment by keachi** [comment]
Sure, this is a CI process.
Ansible Guide on GitHub -> Ansible Guide on GitLab -> CI -> docs.adfinis-sygroup.ch.

It would be much nicer if the documentation is in the defaults/main.yml, else somebody forgets to update the readme all the time. But we need to make a CI process to get that stuff into the readme, else the Ansible Galaxy hasn't a nice view.

**Review by fujexo — APPROVED** [review]
LGTM, thanks for the submission!
